package.json

```
@@ -55,74 +55,75 @@
     "@types/mz": "0.0.32",
     "@types/node": "^10.17.17",
     "@types/request": "^2.48.4",
-    "@types/rimraf": "^2.0.3",
+    "@types/rimraf": "^2.0.4",
     "@types/rwlock": "^5.0.2",
+    "@types/semver": "^7.1.0",
     "@types/shell-quote": "^1.6.1",
     "@types/sinon": "^7.5.2",
     "@types/sinon-chai": "^3.2.3",
     "@types/tmp": "^0.1.0",
-    "balena-sync": "^10.0.0",
+    "balena-sync": "^10.2.0",
     "blinking": "~0.0.3",
     "bluebird": "^3.7.2",
     "body-parser": "^1.19.0",
     "buffer-equal-constant-time": "^1.0.1",
     "chai-as-promised": "^7.1.1",
     "chai-events": "0.0.1",
-    "chokidar": "^3.0.0",
+    "chokidar": "^3.3.1",
     "coffee-loader": "^0.9.0",
     "coffeescript": "^1.12.7",
     "common-tags": "^1.8.0",
     "copy-webpack-plugin": "^4.6.0",
     "dbus-native": "^0.2.5",
     "deep-object-diff": "^1.1.0",
     "docker-delta": "^2.2.9",
-    "docker-progress": "^3.0.3",
-    "docker-toolbelt": "^3.3.7",
+    "docker-progress": "^3.0.5",
+    "docker-toolbelt": "^3.3.8",
     "duration-js": "^4.0.0",
     "event-stream": "3.3.4",
     "express": "^4.17.1",
     "fork-ts-checker-webpack-plugin": "^0.5.2",
-    "fp-ts": "^2.0.5",
+    "fp-ts": "^2.5.3",
     "husky": "^4.2.3",
-    "io-ts": "^2.0.1",
+    "io-ts": "^2.1.2",
     "io-ts-reporters": "^1.0.0",
     "istanbul": "^0.4.5",
-    "json-mask": "^0.3.8",
+    "json-mask": "^0.3.9",
     "knex": "^0.15.2",
     "lint-staged": "^10.0.8",
-    "livepush": "^2.0.2",
+    "livepush": "2.1.0",
     "lockfile": "^1.0.4",
     "lodash": "^4.17.15",
     "log-timestamp": "^0.1.2",
     "memoizee": "^0.4.14",
-    "mixpanel": "^0.10.1",
-    "mkdirp": "^0.5.1",
+    "mixpanel": "^0.10.3",
+    "mkdirp": "^0.5.3",
     "mocha": "^5.2.0",
-    "morgan": "^1.9.1",
+    "morgan": "^1.10.0",
     "mz": "^2.7.0",
     "network-checker": "^0.1.1",
     "pinejs-client-request": "^5.2.0",
     "pretty-ms": "^5.1.0",
     "request": "^2.88.2",
-    "resin-lint": "^3.2.4",
+    "resin-lint": "^3.3.1",
```

Ideally also switch to balena-lint:
```suggestion
    "@balena/lint": "^4.0.1",
```